src/server/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
you don't need body parser only if you receive data from the client, but this is not the case yet